src/components/cc-tile-metrics/cc-tile-metrics.js

```diff
@@ -252,6 +248,9 @@ export class CcTileMetrics extends withResizeObserver(LitElement) {
 
     const panel = this._getCurrentPanel();
 
+    this._cpuChart?.resize();
+    this._memChart?.resize();
+
```

Comment:
question: I tried but I don't understand why it works compared to the `onResize` + `this.*chart.resize()`